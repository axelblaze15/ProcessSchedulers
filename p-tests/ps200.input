1000 1 2147483640 51
1 2147483639 2147483645 50