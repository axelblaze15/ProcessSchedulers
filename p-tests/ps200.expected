1 4294967286 2 2147483647
1000 2147483641 0 2147483640
1 2147483644